src/widgets.js

```diff
@@ -785,6 +781,19 @@ define([
         return el.html();
     }
 
+    function fromRichText(val) {
+        var el = $('<div>');
+        val = val.replace(/(<p>)/ig,"").replace(/<\/p>/ig, "\r\n").replace(/(<br ?\/?>)/ig,"\n").replace('&nbsp;', ' ').replace('\n', ' ');
```

Comment:
Why replace with `\r\n` and then later replace `\n` with space? Seems like you could end up with `\r` in some places.